src/Raven.Server/Documents/DocumentDatabase.cs

```diff
@@ -275,7 +276,7 @@ public void Initialize(InitializeOptions options = InitializeOptions.None)
                 PeriodicBackupRunner = new PeriodicBackupRunner(this, _serverStore);
 
                 _addToInitLog("Initializing IndexStore (async)");
-                _indexStoreTask = IndexStore.InitializeAsync(record);
+                _indexStoreTask = IndexStore.InitializeAsync(record, _addToInitLog);
```

Comment:
Is `_addToInitLog` thread safe?

Reply:
Yes:
https://github.com/ravendb/ravendb/blob/v4.1/src/Raven.Server/Documents/DatabasesLandlord.cs#L595